package.json

```diff
@@ -3,6 +3,18 @@
   "description": "A PWA app that aims to help Buddy and Newbie go through the induction process.",
   "version": "0.1.0",
   "private": true,
+  "scripts": {
+    "start": "react-scripts start",
+    "lint": "eslint ./src/**/*.{ts,tsx}",
+    "lint:fix": "eslint ./src/**/*.{ts,tsx} --fix",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "test:ci": "npm test -- --coverage --watchAll=false",
+    "coverage": "npm test -- --coverage --watchAll=false",
```

Comment:
What is the reason for having 2 commands cloned? Can we maybe get rid of one of them?

Reply:
I will remove the test:ci then